Examples/PythonNative/PythonNative.cpp

```diff
@@ -0,0 +1,91 @@
+#define PY_SSIZE_T_CLEAN
+#include <Python.h>
+#include <Windows.h>
+#include <iostream>
+
+DWORD WINAPI runner(LPVOID lpParam)
+{
+    // Runs the python runner that will listen on the named pipe.
+    Py_Initialize();
+
+    std::cout << "Python version: " << PY_VERSION << std::endl;
+
+    char startFile[] = "C:\\Users\\rchiodo\\source\\repos\\PythonNativeSeparateThread\\runner.py";
+
+    PyObject* startObj = Py_BuildValue("s", startFile);
+    FILE* file = _Py_fopen_obj(startObj, "r+");
```

Comment:
According to the docs at https://docs.python.org/3/c-api/veryhigh.html#c.PyRun_SimpleFile: On Windows, fp should be opened as binary mode (e.g. fopen(filename, "rb")). Otherwise, Python may not handle script file with LF line ending correctly.

Reply:
Thanks. I'll change it.